kube/deployments/smoke-test.yaml

```diff
@@ -15,20 +15,19 @@ spec:
     metadata:
       labels:
         app: origin-web-console-smoke-test
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/scheme: http
     spec:
       containers:
       - name: origin-web-console-smoke-test
-        # see if we find it on dockerhub
         image: benjaminapetersen/origin-web-console-smoke-test:latest
         imagePullPolicy: IfNotPresent
-        # i think the env vars will work here...
         env:
         - name: CONSOLE_URL
-          value: https://10.192.213.145:8443
-        - name: CONSOLE_USER
-          value: bob
-        - name: CONSOLE_PASSWORD
-          value: bob
+          value: https://192.168.1.69:8443
+        - name: TEST_INTERVAL_MINUTES
+          value: '2'
```

Comment:
Did we say 2 mins, or 5 mins for default?

Reply:
default is 5 but I've set the variable here for demo purposes :)